policyengine_us/variables/gov/states/sc/tax/income/sc_state_tax_addback.py

```diff
@@ -0,0 +1,43 @@
+from policyengine_us.model_api import *
+
+
+class sc_state_tax_addback(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "South Carolina State Tax addback"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://dor.sc.gov/forms-site/Forms/SC1040_2022.pdf"
```

Comment:
please add reference to addback description

Reply:
it seems like we need to follow the worksheet from [this](https://dor.sc.gov/forms-site/Forms/SC1040inst_2022.pdf) form - this also depends on whether the person itemized on their return

Reply:
Hi Pavel, we did write our code according the link you send above, we also used the federal 1040 schedule A ([this](https://www.irs.gov/pub/irs-pdf/f1040sa.pdf) form) to see the components of itemized deductions.

Reply:
Sounds good, could just add the [instructions form](https://dor.sc.gov/forms-site/Forms/SC1040inst_2022.pdf) as a second reference so we have it